cmd/internal/shared/init.go

```diff
@@ -30,3 +34,31 @@ func InitializeFallbacks(conf *config.ServiceBase) error {
 	}
 	return nil
 }
+
+// InitializeLogger initializes the logger.
+func InitializeLogger(conf *config.Log) (log.Stack, error) {
+	var (
+		logHandler log.Handler
+		err        error
+	)
+	switch strings.ToLower(conf.Format) {
+	case "json", "console":
+		logHandler, err = log.NewZap(conf.Format)
```

Comment:
`CONSOLE` matches the case condition but ...

Reply:
Good one. Lowercasing this on L44 now.